test/integration.test.ts

```diff
@@ -531,4 +531,18 @@ describe('integration tests', () => {
     expect(createdPlanes).toHaveLength(250);
     expect(queriedPlanes.records).toHaveLength(PAGINATION_DEFAULT_SIZE);
   });
+
+  test('Link is a record object', async () => {
+    const team = await client.db.teams.getOne();
+    const owner = await team?.owner?.read();
```

Comment:
Test fail because we're not actually following the links:

```
{
      team: {
        id: 'rec_c9kfe2gfqmgi3pjav49g',
        labels: [ 'apple', 'banana', 'orange' ],
        name: 'Team fruits',
        owner: { id: 'rec_c9kfe2e7gmf268o1k33g' },
        xata: { version: 0 }
      },
      owner: {
        address: { street: 'Main Street', zipcode: 100 },
        email: 'owner.fruits@example.com',
        full_name: 'Owner of team fruits',
        id: 'rec_c9kfe2e7gmf268o1k33g',
        xata: { version: 0 }
      }
    }
```